ccf_transaction_processor/Makefile

```diff
@@ -75,15 +75,22 @@ clean-build:
 clean-install :
 	rm -rf $(CCFDSTDIR)
 
-config : $(ETCDIR)/cchost.toml $(ETCDIR)/gov.lua
+config : $(ETCDIR)/cchost_start.toml $(ETCDIR)/cchost_join1.toml $(ETCDIR)/gov.lua
 
```

Comment:
Thank you for the question. only reason why I kept separate templates was that when we start these nodes in separate VMs, the extra nodes only need to bother with the join-toml rather than the whole of it.

Reply:
there is no value in two files here. the same config works regardless. so this file just seems to add to the configuration complexity. you can have one file with a [start] section AND a [join] section so all that you want works from a single file

Reply:
Thank you, I agree that for a template, we can keep just one file, and keep expanding that. Except that while expanding the section on [start] will appear no all tomls including the ones that are only going to join.

Reply:
of course.... and when you want to start a network from a different node you'll be able to.